src/edge/edgeManager.ts

```diff
@@ -341,6 +342,10 @@ export class EdgeManager {
                 label: Constants.CSHARP_FUNCTION,
                 description: Constants.CSHARP_FUNCTION_DESCRIPTION,
             },
+            {
+                label: Constants.ACR_MODULE,

```

Comment:
instead of have ACR_MODULE on the same level with EXISTING_MODULE, how about make it sub of EXISTING_MODULE, user could either input or select form ACR?

Reply:
I am afraid existing UI controls provided by VS Code don't allow the coexistence of an input box and a quick pick item.